specification/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/attestation.json

```diff
@@ -525,6 +568,45 @@
         }
       }
     },
+    "OpenIDConfigurationResponse": {
+      "type": "object",
+      "description": "The response to the OpenID metadata description document API",
+      "properties": {
+        "response_types_supported": {
+          "description": "Types supported in the OpenID metadata API",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "id_token_signing_alg_values_supported": {
+          "description": "List of the supported signing algorithms",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "revocation_endpoint": {
+          "description": "Revocation endpoint",
+          "type": "string"
+        },
+        "issuer": {
+          "description": "Issuer tenant base endpoint",
+          "type": "string"
+        },
+        "jwks_uri": {
+          "description": "The URI to retrieve the signing keys",
+          "type": "string"
+        },
+        "claims_supported": {
+          "description": "Set of claims supported by the OpenID metadata endpoint",
```

Comment:
Shouldn't these all be camelCase to avoid another LintDiff failure in the future?

Reply:
The LintDiff errors should be suppressed or an exception added because of two reasons:

- the definition of the openid configuration endpoint follows the Open Id spec, see https://openid.net/specs/openid-connect-core-1_0.html
- switching from the "snake" case to the camel case would be customer breaking changes

Reply:
Received the swagger suppression approval and added the suppression to the readme file